src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectgrid/view/ProjectGridView.kt

```diff
@@ -27,6 +27,7 @@ class ProjectGridView : Fragment() {
     init {
         importStylesheet<ProjectGridStyles>()
         importStylesheet<DefaultStyles>()
+        importStylesheet<MainScreenStyles>()
```

Comment:
same with styles here

Reply:
should be in AppStyles

Reply:
So should I put `MainScreenStyles.belowMenuBar` in `AppStyles` as well?

Reply:
I think so